### PR TITLE
[Catalog Item]  update action button alignment when title wraps

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] update action button alignment when title wraps

## Screenshots

|  Before  |  After  |
|--------|--------|
|Screen Shot 2021-07-23 at 2 14 13 PM|Screen Shot 2021-07-23 at 2 14 06 PM|

## Testing in `sage-lib`

Visit the catalog item page. (You may need to update the text to wrap)
Verify that the action buttons don't stretch to match the height of the wrapped title

## Testing in `kajabi-products`

1. (LOW)  Description of the change and its impact with QA as the audience.
   - [ ] Product Outlines view

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_page_heading.scss [hunk 1]
```diff
@@ -122,6 +122,10 @@ $-page-heading-image-height-mobile: rem(120px);
     flex-wrap: nowrap;
     justify-content: flex-end;
   }
+
+  .sage-btn {
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/components/_page_heading.scss:126]: Could this go in `sage-btn` instead?

**Review by philschanely — APPROVED** [review]
On small suggestion regarding placement for potential leakage. Otherwise 👍

File: packages/sage-assets/lib/stylesheets/components/_page_heading.scss (hunk 1 above)
Comment (QuintonJason) [reply packages/sage-assets/lib/stylesheets/components/_page_heading.scss:126]: Can do

**Review by monicawheeler — APPROVED** [review]

**Review by pixelflips — APPROVED** [review]
LGTM! 🔥

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM, fix is working great

**Pull request merged**